roles/podman/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+podman_user: "{{ receptor_user | default('podman') }}"
```

Comment:
I don't think a defaults file should make use of variables from a different role.  These should be in my opinion just 'podman', and callers that are actually using both should be responsible for making them the same thing, if that is actually what is desired.